### PR TITLE
build(property-dds): convert to fluid-tsc

### PR DESCRIPTION
**Pull request by DLehenbauer**
- fix explicit paths for full ESM
- address lodash imports
- various corrections from more strict tsc options with tests
   including testing LZ4PropertyTree not previously tested.

## remove mocha-ts scripts
test:mocha-ts* are not usable per several issues with ts-node (10.9.2 or earlier):
- https://github.com/TypeStrong/ts-node/issues/2000 no tsconfig extends multiple files support
- https://github.com/TypeStrong/ts-node/issues/1967 lack of .ts + ESM understanding
- failure to resolve .js extensions using CommonJS

### CODE REVIEW

File: experimental/PropertyDDS/packages/property-dds/src/test/rebasing.spec.ts [hunk 1]
```diff
@@ -463,7 +467,7 @@ describe("PropertyDDS", () => {
 			});
 
 			describe("Randomized Tests", () => {
-				const count = 100;
+				const count = 1;
```
Comment (jason-ha) [inline experimental/PropertyDDS/packages/property-dds/src/test/rebasing.spec.ts:470]: @DLehenbauer, do you want to keep this test count reduction? Temp for local testing?

Comment (DLehenbauer) [reply experimental/PropertyDDS/packages/property-dds/src/test/rebasing.spec.ts]: Reverted.  Thank you for noticing.

**Comment by DLehenbauer** [comment]
@jason-ha - Looks good to me.  Thank you!

**Review by jason-ha — APPROVED** [review]

**Comment by mp3por** [comment]
WHY IS THIS NOT AVAILABLE THROUGH NPM ?

**Pull request merged**